### DISCUSSION TITLE
Sheriff Mod

### DISCUSSION THREAD

**Discussion by Snowad14** [Woodi-dev/Among-Us-Sheriff-Mod#3] [Q&A] [answered]
Thx Woodi-dev
For this mod, first question: How long did you take to create this mod and second question: Are you going to create another mod? I love this mod and it works perfectly

**Comment by Woodi-dev** [comment, marked as answer]
Thanks, share it and play it with your friends. I will keep this mod up to date. It took me just a few days to create this mod. However, the reverse engineering of the game was the most time-consuming part (some weeks). I am going to release other mods for the game if the community likes them.

**Reply by Sloppy007** [reply]
Hi Woodi-dev i was wondering if you could teach me how you make mods for Among Us

**Reply by Petitcu** [reply]
Hi Woodi-dev, if u want we are 3 devs who can help you in mod dev, if you are interested please tell us

**Reply by KingsMMA** [reply]
I'm also interested in learning how to make mods, if you are willing to post a tutorial.

The community would really appreciate it if you did that, as it would mean if you wanted a mod for something, it would be a lot easier to find it

**Reply by Snowad14** [reply]
@Petitcu if you know c # mod it shouldn't be too hard add me on discord Raphael # 0666 if you want me to show you how to setup the modding environment

**Reply by o4pixel** [reply]
can anyone tell me how to make among us roles 
(well the sad thing is i don't have discord)but please i mod among us 
with textures but i want to learn this :(

**Reply by tartsharpcobra** [reply]
Post a tutorial!

**Reply by BOU-zts** [reply]
keep getting a black screen, in 1.2 after an online game started, in 1.21 at the end of the online game

**Comment by floydgeorge38** [comment]
Hi

**Reply by floydgeorge38** [reply]
Huge

**Comment by tartsharpcobra** [comment]
I love this mod!

**Comment by BethBurnsi** [comment]
This is a very good mod. But currently there is a new mod on [AmongUsComBo](https://amonguscombo.com/)
